src/mixins/name-request-mixin.ts

```diff
@@ -67,6 +79,21 @@ export default class NameRequestMixin extends Mixins(DateMixin) {
     )
   }
 
+  /**
+  * Returns True if the Name Request data is valid for New NR.
+  * @param nr the name request response payload
+  * */
+  isNewNrValid (nr: any): boolean {
+    return Boolean(nr &&
+      nr.state &&
+      nr.expirationDate &&
+      !!this.getNrApprovedName(nr) &&
+      nr.nrNum &&
+      nr.requestTypeCd &&
+      [NameRequestTypes.NEW].includes(nr.request_action_cd)
```

Comment:
I don't think we use a "new" NR for changing a business name. I believe we use a "change" NR, like this:

![image](https://user-images.githubusercontent.com/10002889/180847052-d24218ed-c6d3-4004-857a-60078d264f71.png)

Please confirm with PO.

Reply:
This PR is blocked until this requirement is confirmed.

Reply:
Confirmed by Linda that a "new" name request cannot be used in a firm correction (and not a "conversion" NR either). Only a "change of name" NR can be used.